disk/metadata.csv

```diff
@@ -1,13 +1,15 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 system.disk.free,gauge,,byte,,The amount of disk space that is free.,1,system,disk free,
-system.disk.in_use,gauge,,percent,,The amount of disk space in use as a percent of the total.,-1,system,disk in use,
+system.disk.in_use,gauge,,fraction,,The amount of disk space in use as a fraction of the total.,-1,system,disk in use,
```

Comment:
```suggestion
system.disk.in_use,gauge,,fraction,,The amount of disk space in use as a percent of the total.,-1,system,disk in use,
```

As confusing as it is, the fraction actually shows as a percent on the frontend. 

https://a.cl.ly/ApugYGy2
https://a.cl.ly/DOuKDoGo

Because the fraction unit in catalog.csv is in the percent family with a scale factor of 100